arangod/RocksDBEngine/RocksDBKey.cpp

```diff
@@ -149,6 +150,35 @@ void RocksDBKey::constructMdiIndexValue(uint64_t indexId,
   TRI_ASSERT(_buffer->size() == keyLength);
 }
 
+void RocksDBKey::constructVectorIndexValue(uint64_t indexId,
+                                           std::size_t listNumber) {
+  _type = RocksDBEntryType::VectorVPackIndexValue;
+  // Key contains: indexId(uint64_t) + listNumber(size_t)
+  size_t keyLength = sizeof(uint64_t) + sizeof(std::size_t);
```

Comment:
We are using `std::size_t` for nLists because faiss uses the same. So if the architecture changes that should change as well. 
Check in `RocksDBInvertedLists` class in `RocksDBVectorIndex.cpp` which inherits `nLists` from `faiss::InvertedLists`. Also the key is constructed in `add_entries` which accepts only `size_t` as `listNumber` value.

Reply:
Sorry for misunderstanding. Resolved after talking, will write `uint64_t` in RocksDB